client/multi-currency-analytics/index.js

```diff
@@ -5,9 +5,13 @@
 import { addFilter } from '@wordpress/hooks';
 import { __ } from '@wordpress/i18n';
 
-const customerCurrencies = wcSettings.customerCurrencies.sort( ( a, b ) => {
-	return a.label < b.label ? -1 : 1;
-} );
+const getCustomerCurrencies = () => {
+	return (
+		wcSettings.customerCurrencies.sort( ( a, b ) => {
+			return a.label < b.label ? -1 : 1;
+		} ) ?? []
```

Comment:
@dmallory42 I've tried testing the PR on my JN test site and encountered the following (not sure whether it's related to the current change).

1. Created 2 orders on the test site, one with USD & the other with EURO.
2. Under Analytics -> Orders, I am able to see both the orders with the correct currency
3. Under Advanced Filters -> Customer currency, I'm not seeing the option to choose a currency. Could be related to passing an empty array here?

![Screenshot](https://d.pr/i/pfs89U+)

Reply:
This could be related to catching - there can be a short delay (5 mins) between a currency being used for the first time and displaying in the analytics section. Could you check again to ensure this isn't the case here?

I will also update the logic so the store default currency is always on the list even when no orders have been made on the store, since this feels a bit more like the expected behaviour.

Reply:
Yeah, it was caching. I am able to see the currency drop-down now.

Reply:
Awesome, good to know. 👍

Reply:
Created a follow-up issue to always show the default currency: https://github.com/Automattic/woocommerce-payments/issues/4479 🙂